core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadHandler.java

```diff
@@ -87,20 +85,14 @@ public void handle(String s, Request request, HttpServletRequest httpServletRequ
       if (!s.startsWith(mS3Prefix)) {
         return;
       }
-      // Build log message capturing the request details
-      // TODO(czhu): Move this to a separate logging handler
-      StringBuilder sb = new StringBuilder();
-      sb.append("Alluxio S3 API received ");
-      sb.append(request.getMethod());
-      sb.append(" request: URI=");
-      sb.append(s);
-      if (request.getQueryString() != null) { sb.append("?").append(request.getQueryString()); }
-      sb.append(" User=");
+      if (!request.getMethod().equals("POST") || request.getParameter("uploadId") == null) {
+        return;
+      } // Otherwise, handle CompleteMultipartUpload
       final String user;
       try {
         // TODO(czhu): support S3RestServiceHandler.getUserFromSignature()
         //             Ideally migrate both to S3RestUtils and make them static
-        user = S3RestServiceHandler.getUserFromAuthorization(
+        user = S3AuthenticationFilter.getUserFromAuthorization(
```

Comment:
So S3RestServicehandler is no longer used?

Reply:
just migrate the methods related to Authentication into the filter, maybe they should be migrated to S3RestUtils?